src/core/server/metrics/metrics_service.ts

```diff
@@ -71,10 +77,15 @@ export class MetricsService
     return this.service;
   }
 
-  private async refreshMetrics() {
+  private async refreshMetrics(thresholdMs: number) {
     const metrics = await this.metricsCollector!.collect();
     const { message, meta } = getEcsOpsMetricsLog(metrics);
+    const { message: eventLoopThresholdMessage } = getEventLoopThresholdLog(metrics, thresholdMs);
+
     this.opsMetricsLogger.debug(message!, meta);
+    if (eventLoopThresholdMessage) {
```

Comment:
I think we may need to throttle this or else it may be logged very frequently (however often metrics are refreshed which I think is 30s by default).